iot-hub/Samples/device/PnpDeviceSamples/PnpConvention/PnpConvention.cs

```diff
@@ -202,95 +195,145 @@ public static string CreateComponentPropertyPatch(string componentName, IDiction
-        /// </code>
-        ///
-        /// For a component-level property update, the patch is in the format:
-        /// <code>
-        /// {
-        ///   "sampleComponentName": {
-        ///     "__t": "c",
-        ///     "samplePropertyName": {
-        ///       "value": 20,
-        ///       "ac": 200,
-        ///       "av": 5,
-        ///       "ad": "The update was successful."
-        ///     }
-        ///   }
-        /// }
-        /// </code>
-        /// </remarks>
-        /// <param name="propertyName">The property name, as defined in the DTDL interface.</param>
-        /// <param name="unserializedPropertyValue">The unserialized property value, in the format defined in the DTDL interface.</param>
-        /// <param name="ackCode">The acknowledgment code from the device, for the embedded value property update.</param>
-        /// <param name="ackVersion">The version no. of the service-initiated read-write property update.</param>
-        /// <param name="unserializedAckDescription">The serialized description from the device, accompanying the embedded value property update.</param>
-        /// <param name="componentName">The name of the component in which the property is defined. Can be null for property defined under the root interface.</param>
-        /// <returns>The property patch for embedded value property updates for read-write properties.</returns>
-        public static string CreatePropertyEmbeddedValuePatch(
+        /// <param name="propertyName">The name of the property to report.</param>
+        /// <param name="propertyValue">The unserialized property value.</param>
+        /// <param name="ackCode">The acknowledgement code, usually an HTTP Status Code e.g. 200, 400.</param>
+        /// <param name="ackVersion">The acknowledgement version, as supplied in the property update request.</param>
+        /// <param name="ackDescription">The acknowledgement description, an optional, human-readable message about the result of the property update.</param>
+        /// <returns>A serialized json string response.</returns>
+        public static string CreateWritablePropertyResponse(
             string propertyName,
-            object unserializedPropertyValue,
+            object propertyValue,
             int ackCode,
             long ackVersion,
-            string unserializedAckDescription = default,
-            string componentName = default)
+            string ackDescription = null)
         {
             if (string.IsNullOrWhiteSpace(propertyName))
             {
                 throw new ArgumentNullException(nameof(propertyName));
             }
-            if (unserializedPropertyValue == null)
+
+            var response = new Dictionary<string, WritablePropertyResponse>
+            {
+                { propertyName, new WritablePropertyResponse(propertyValue, ackCode, ackVersion, ackDescription) },
+            };
+
+            return JsonConvert.SerializeObject(response);
```

Comment:
why not have this method call the overload on L234?

Reply:
Because I worked backwards and forgot to update this one too. :p